### PR TITLE
Mendix Log Node Descriptions

### PR DESCRIPTION
**Pull request by MarkvanMents**
There is no current description of the log nodes used by Mendix when writing log message

Lenneart will help to provide this information for publication.

### CODE REVIEW

File: content/refguide/logging.md [hunk 1]
```diff
@@ -37,10 +37,52 @@ This is the most verbose logging level, and can be used if you want even more fi
...
+| ConnectionBus_Security | Information regarding access rights needed to access the database. |
+| ConnectionBus_Synchronize | Deprecated: This is a legacy node. |
+| ConnectionBus_Update | All information related to the update of data in the database. Incoming storage requests, the executed statements and issues encountered during storage. |
+| ConnectionBus_Validation | Information related modification of the existing database, and database migration. |
+| Connector | |
+| Core | Logs messages from the core runtime. This can be startup of the runtime, version of the runtime, license being used and issues related to interpreting the model. |
+| DataStorage_QueryHandling | Logs messages related to the queries that are being executed. |
+| DataStorage_QueryPlan | Query execution plan information for installations (currently only supported for PostgreSQL databases). |
+| DocumentExplorer | Logs messages related to the templating engine that generates documents. |
```
Comment (grootjans) [inline content/refguide/logging.md:62]: This log node is called 'DocumentExporter'

File: content/refguide/logging.md [hunk 2]
```diff
@@ -37,10 +37,52 @@ This is the most verbose logging level, and can be used if you want even more fi
...
+| FileDocumentSizesPopulateJob | Logs messages for a background job that populates the file-size field in the database for documents that do not have that field filled (used during legacy migration). |
+| I18NProcessor | Logs messages related to translation of the app. |
+| JSON | JSON messages from the Mendix Client to the Runtime Server. See [JSON](#json), below, for more information |
+| Jetty | Logs messages from the internal Jetty webserver that handles HTTP requests between the runtime and the outside world. |
+| LocalFileSystemStore | Logs messages related to file handling if you are using local file system as your file store. |
+| Logging | Logs messages related to the logging framework used by Mendix. |
+| MicroflowDebugging | Log messages related to the status of the microflow debugger, e.g. connection status, incoming and outgoing requests, etc. |
```
Comment (grootjans) [inline content/refguide/logging.md:69]: This should be 'MicroflowDebugger'

**Comment by MarkvanMents** [comment]
@grootjans - are you happy for me to publish this as it is, and we can work on describing the rest of the nodes in another 'sprint'?

**Comment by MarkvanMents** [comment]
Agreed to publish with Robbert Jan.

**Pull request merged**